packages/shared-core/src/automations/steps/bash.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  AutomationActionStepId,
+  AutomationCustomIOType,
+  AutomationFeature,
+  AutomationIOType,
+  AutomationStepDefinition,
+  AutomationStepType,
+} from "@budibase/types"
+
+export const definition: AutomationStepDefinition = {
+  name: "Bash Scripting",
+  tagline: "Execute a bash command",
+  icon: "JourneyEvent",
+  description: "Run a bash script",
+  type: AutomationStepType.ACTION,
+  internal: true,
+  features: {
+    [AutomationFeature.LOOPING]: true,
+  },
+  stepId: AutomationActionStepId.EXECUTE_BASH,
+  inputs: {},
+  schema: {
+    inputs: {
```

Comment:
Would not be enough moving inputs and outputs? Will we be using anything else in the frontend?

Reply:
cc @deanhannigan, I believe the big win from this would be testing.